cmake/configs/posix_rpi_common.cmake

```diff
@@ -74,6 +74,7 @@ set(config_module_list
 	#
 	# PX4 drivers
 	#
+	drivers/linux_sbus
```

Comment:
Yes，I have tested it  under ubuntu，debian ，raspbian , fedora.
I found linux systems support nonstandard serial communication but this driver dosent work under a virtual machine if the virtual machine was running under mac os